### DISCUSSION TITLE
Now that we have UAnimInstance::NativeThreadSafeUpdateAnimation in 5.0, is it make sense to use FAnimInstanceProxy any more?

### DISCUSSION THREAD

**Discussion by ameaninglessname** [Sixze/ALS-Refactored#171] [Q&A] [answered]
[And this doc from epic looks absolutely outdated...](https://docs.unrealengine.com/5.0/en-US/animation-optimization-in-unreal-engine/)

**Comment by Sixze** [comment, marked as answer]
`FAnimInstanceProxy` is still there and you can still use it, but `UAnimInstance::NativeThreadSafeUpdateAnimation` is just an easier way to use multithreading. It's also more dangerous because you're mixing code for different threads in the same `UAnimInstance` class, and you have to be very careful with that.
